### ISSUE TITLE
Unable to checkout the code under windows

### ISSUE THREAD

**Issue by louyihua** [openshift/origin#14664]
The vendoring of cluster-capacity in Origin prevents Origin being checked-out under Windows, since this package has files/directories with the forbidden-character `:` in filename like `cmd/cluster-capacity/go/src/github.com/kubernetes-incubator/cluster-capacity/vendor/github.com/docker/distribution/contrib/docker-integration/generated_certs.d/localregistry:5440`.
Although Origin itself is not targeted for Windows, I think there are many people who need to edit or view codes under Windows. So could someone help to solve this problem?

**Comment by aveshagarwal** [comment]
I am looking into it.

**Comment by aveshagarwal** [comment]
@louyihua could you test https://github.com/openshift/origin/pull/14683 and see if that works or are there any more files/dirs with issues in windows?

**Comment by louyihua** [comment]
@aveshagarwal
Your PR works for the `cluster-capacity` directory.
But there is another instances of `generated_certs.d` in the `service-catalog` and the checkout is blocked by that one. You may need to remove that directory there as well.

**Comment by aveshagarwal** [comment]
@louyihua Could test again as I updated the pr for service-catalog too.

**Comment by louyihua** [comment]
@aveshagarwal This update works for `service-catalog`.
But I still can't fully checkout your branch due to issues under `vendor` directory that seems to be solved through a UPSTREAM import. I think your PR should work if it is merged into current master.
